jukebox/Interacting_with_Jukebox.ipynb

```diff
@@ -100,20 +135,126 @@
         "colab": {}
       },
       "source": [
-        "model = \"5b_lyrics\" # or \"1b_lyrics\"     \n",
+        "model = \"5b_lyrics\" # or \"1b_lyrics\"\n",
         "hps = Hyperparams()\n",
         "hps.sr = 44100\n",
         "hps.n_samples = 3 if model=='5b_lyrics' else 8\n",
-        "hps.name = 'samples'\n",
+        "# Specifies the directory to save the sample in.\n",
+        "# We set this to the Google Drive mount point.\n",
+        "hps.name = '/content/gdrive/My Drive/samples'\n",
         "chunk_size = 16 if model==\"5b_lyrics\" else 32\n",
         "max_batch_size = 3 if model==\"5b_lyrics\" else 16\n",
         "hps.levels = 3\n",
         "hps.hop_fraction = [.5,.5,.125]\n",
         "\n",
         "vqvae, *priors = MODELS[model]\n",
```

Comment:
just as a though: since `Downloading from gce` (Downloading the model, I assume..?) takes quite a long time, especially for the 5b -models, it might make sense to split those two. So you can edit the `Hyperparams ` without reloading the model each time :)